guides/inference-scheduling/helmfile.yaml.gotmpl

```
@@ -107,8 +106,6 @@ releases:
       - ms-inference-scheduling/values_cpu.yaml
     {{- else if eq .Environment.Name "digitalocean" }}
       - ms-inference-scheduling/digitalocean-values.yaml
-    {{- else if eq .Environment.Name "xpu" }}
-      - ms-inference-scheduling/values_xpu.yaml
```

Revert, why remove XPU support
```suggestion
    {{- else if eq .Environment.Name "xpu" }}
      - ms-inference-scheduling/values_xpu.yaml
```